layout/mainmenu_settings.xml

```diff
@@ -388,6 +388,15 @@
 								>
 									<Label class="settings-nav__subitem-label" text="#Settings_WeaponSelection_Title" />
 								</RadioButton>
+								
+								<RadioButton
+									id="JumpStatsRadio"
+									class="settings-nav__subitem"
+									group="SettingsSubNav"
+									onactivate="MainMenuSettings.navigateToSubsection('InterfaceSettings', 'JumpStatsSubSection')"
+								>
+									<Label class="settings-nav__subitem-label" text="Jump Stats" />
```

Comment:
Unlocalized?

Reply:
Should probably be `#Settings_JumpStats_Title` right?

Reply:
yep, good look. Missed this file completely when I went through the localization stuff